pytest/test_profile.py

```diff
@@ -71,8 +71,8 @@ def testProfileReset(env):
 	env.expect('RG.PYPROFILE', 'RESET', sessionId).ok()
 	env.expect('RG.PYPROFILE', 'STATS', sessionId).error()
 
-@gearsTest(skipOnCluster=True)
+@gearsTest(skipOnCluster=True, envArgs={'moduleArgs': 'CreateVenv 1'})
 def testPyDumpSessions(env):
 	env.expect('RG.PYEXECUTE', "GB('CommandReader').register(trigger='test')", 'REQUIREMENTS', 'redis==3.5.3').ok()
 	sessionId = getSession(env)
-	env.expect('RG.PYDUMPSESSIONS').equal([['ID', sessionId, 'sessionDescription', None, 'refCount', 1, 'linked', 'primary', 'dead', 'false', 'requirementInstallationNeeded', 0, 'requirements', ['redis==3.5.3'], 'registrations', ['0000000000000000000000000000000000000000-2']]])
+	env.expect('RG.PYDUMPSESSIONS').equal([['ID', sessionId, 'sessionDescription', None, 'refCount', 1, 'linked', 'primary', 'dead', 'false', 'requirementInstallationNeeded', 1, 'requirements', ['redis==3.5.3'], 'registrations', ['0000000000000000000000000000000000000000-1']]])
```

Comment:
Any way to change this to redis==4.3.1?  Also, there are other dependencies it comes with.